feat(ffe-tabs-react): add props thin and className to TabGroup

henrikhermansen opened the pull request:
This commit adds two new props to the `TabGroup` component.
Bool `thin` adds the `--thin` modifier to remove top and bottom padding.
String `className` adds given css classes to the tab group.

See #203 for screen shots

selbekk reviewed (dismissed):
Ser bra ut! Kjører du endringene dine gjennom prettier også?

Commit:
feat(ffe-tabs-react): add props thin and className to TabGroup

This commit adds two new props to the `TabGroup` component.
Bool `thin` adds the `--thin` modifier to remove top and bottom padding.
String `className` adds given css classes to the tab group.

Commit:
chore(ffe-tabs-react): bump peerDependency of ffe-tabs

Bump the peer dependency of `ffe-tabs` to make sure consumers have the new
class `ffe-tab-button-group` and its modifier available.

henrikhermansen commented:
Oh my. I thought prettier was still prettifying my edited lines. Maybe we should re-enable that?
Anyways, I've made some new, prettified commits now.
